app/controllers/security_events_controller.rb

```diff
@@ -26,6 +32,21 @@ def show
     @security_event = security_event
   end
 
+  def search
+    email = params[:email]
+
+    if email.present?
+      if (user = User.find_by(email: email))
```

Comment:
Do we want to confine this to the groups the user is in?

Reply:
the "all" page is for admins only, so I think it's fine to show across all groups